example/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const bootstrap = require('hof-bootstrap');
+
+bootstrap({
+  views: false,
+  fields: false,
```

Comment:
nice one - and while you're at it, it shouldn't throw an error if there isn't a views directory